### PR TITLE
Updates to pytorch-aarch64 and tensorflow-aarch64 docker builds.

### PR DESCRIPTION
**Pull request by nSircombe**
- Updates OS to Ubuntu 20.04
 - Updates python environment to 3.8 with some package updates
 - Removes TF 1.x build option
 - Removes ArmPL dependencies (oneDNN build defaults to ACL)
 - Adds obj. detection example
 - Minor re-structuring of examples folders
 - Various corrections and updates to README files

Change-Id: Iefc770d757cbc024ea820562a146f23a65e97ac0

### CODE REVIEW

File: docker/tensorflow-aarch64/scripts/build-numpy.sh [hunk 1]
```diff
@@ -30,23 +30,15 @@ git clone ${src_host}/${src_repo}.git
...
-if [[ $ONEDNN_BUILD = "openblas" ]]; then
-  export BLAS_LIB='openblas'
-  export BLAS_DIR=$OPENBLAS_DIR
-  export BLAS_LDFLAGS=""
-else
-  export BLAS_LIB='armpl_lp64'
-  export BLAS_DIR=$ARMPL_DIR
-  export BLAS_LDFLAGS="-lgfortran"
-fi
+# OpenBLAS libraries are used to build numpy
```
Comment (cfRod) [inline docker/tensorflow-aarch64/scripts/build-numpy.sh:33]: Licence header to 2020-2021

File: docker/tensorflow-aarch64/scripts/build-benchmarks.sh [hunk 2]
```diff
@@ -24,15 +24,6 @@ cd /home/$DOCKER_USER
...
-if [[ $tf_id == '1' ]]; then
-  src_branch=cnn_tf_v1.15_compatible
-elif [[ $tf_id == '2' ]]; then
-  src_branch=cnn_tf_v2.1_compatible
-else
-  echo 'Invalid TensorFlow version when installing benchmarks'
-  exit 1
-fi
-
+src_branch=cnn_tf_v2.1_compatible
```
Comment (cfRod) [inline docker/tensorflow-aarch64/scripts/build-benchmarks.sh:27]: Licence header 2020-2021

**Comment by cfRod** [comment]
Looks good

File: docker/tensorflow-aarch64/Dockerfile [hunk 3]
```diff
@@ -268,22 +249,18 @@ ENV VENV_DIR=/home/$DOCKER_USER/python3-venv
 COPY --chown=$DOCKER_USER:$DOCKER_USER --from=tensorflow-tools $VENV_DIR /home/$DOCKER_USER/python3-venv
 ENV PATH="$VENV_DIR/bin:$PATH"
 
-# Build Bazel
-COPY scripts/build-bazel.sh $PACKAGE_DIR/.
-RUN $PACKAGE_DIR/build-bazel.sh
-ENV PATH=$PACKAGE_DIR/bazel/output:$PATH
+# Get Bazel binary for AArch64
+COPY scripts/get-bazel.sh $PACKAGE_DIR/.
+RUN $PACKAGE_DIR/get-bazel.sh
```
Comment (elfringham) [inline docker/tensorflow-aarch64/Dockerfile:254]: This file get-bazel.sh seems to be missing from this repository.

Comment (nSircombe) [reply docker/tensorflow-aarch64/Dockerfile:254]: Yep, well spotted. There was slight mix-up. It is there but erroneously named `get-onednn.sh`.

There's a fix up now: https://github.com/ARM-software/Tool-Solutions/commit/1b0e58ba93ca19fcf8f48af5c135111c5cc897a7

**Pull request merged**